src/pages/departures.tsx

```diff
@@ -23,17 +24,23 @@ function DeparturesContent({ autocompleteFeatures }: DeparturesContentProps) {
       <input type="text" onChange={(e) => setQuery(e.currentTarget.value)} />
 
       {autocompleteFeatures.map((f, i) => (
-        <div key={i}>{f.name}</div>
+        <div key={i}>
```

Comment:
this should really be a unique ID. Using the index here isn't actually recommended. I can talk at length about why in a call if you want 😅

Reply:
It was `i` in the example just to get eslint stop complaining but it shouldn't been in the first place. Just me being lazy (sorry)

Reply:
I'm aware 😛 Was going to see if the result from the endpoint had an ID, which I assume it has.

Reply:
Same with the key for the result list.

Reply:
It does!

Reply:
https://github.com/AtB-AS/planner-web/blob/fec409dddaa0a5d2cff39e60ba4ff9b4fd306e37/src/components/search/index.tsx#L81-L83

How should these keys be handled optimally?

Reply:
Hmm. I think in this case index is the best we got. But you could wrap the entire thing in a span and let that have the key also 🤔